### PR TITLE
Boost.config@1.86.0

### PR DESCRIPTION
**Pull request by Vertexwahn**
Boost.config@1.86.0

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @wep21, modules you maintain (boost.config) have been updated in this PR. Please review the changes.

File: modules/boost.config/1.86.0/overlay/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,24 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "boost.integer",
```
Comment (wep21) [inline modules/boost.config/1.86.0/overlay/BUILD.bazel:4]: This BUILD file seems to be the file for `boost.integer`.

Comment (Vertexwahn) [reply modules/boost.config/1.86.0/overlay/BUILD.bazel]: Fix it!

**Review by wep21 — APPROVED** [review]
LGTM with green ci.

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

File: modules/boost.config/1.86.0/overlay/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cc_library(
+    name = "boost.config",
+    hdrs = glob(
+        [
+            "include/**/*.hpp",
+            "include/**/*.h",
+        ],
+    ),
+    features = [
+        "parse_headers",
```
Comment (fmeum) [inline modules/boost.config/1.86.0/overlay/BUILD.bazel:12]: Some of the headers are not meant to be included and should be moved into `textual_hdrs`.

**Pull request closed without merging**